ddprofiler/app/src/main/resources/spatial_patterns.json

```diff
@@ -1,2 +1,16 @@
 {
+  "geoCoordinate": [
+    "^POINT\\(([-+]?[0-9]*\\.?[0-9]+)[ ,]+([-+]?[0-9]*\\.?[0-9]+)\\)$",
+    "^\\((-?\\d+(\\.\\d+)?(.)?°)[ ,]+(-?\\d+(\\.\\d+)?(.)?°)\\)$"
```

Comment:
No, it can't because I didn't find any coordinate with that pattern. But it is highly likely to exist as well, so should I modify it? What do you think?

Reply:
yes, i think we should add it

Reply:
Alright, I have modified it.